frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -60,9 +66,34 @@ const LineAreaBarChart = ({
     chartSize,
   );
 
+  const keys = series
+    .map(singleSeries => {
+      if (singleSeries.seriesKey) {
+        return singleSeries.seriesKey;
+      }
+
+      return formatStaticValue(singleSeries.name, {
```

Comment:
More explanation for this line here
https://github.com/metabase/metabase/pull/26211/files#r1012746149